Makefile

```diff
@@ -0,0 +1,23 @@
+.PHONY: clean package package-deps package-source package-upload package-wheel
```

Comment:
Okay, I know I'm being totally OCD here, but is there a reason the PHONY list is in a different order than the actual directives? 😖

Reply:
Only cause I copy-pasted it! But I'll fix it so it doesn't bother you or the next developer to notice haha (we do tend to be a detail-focused lot!)